src/sirinternal.c

```diff
@@ -613,12 +628,14 @@ bool _sir_logv(sir_level level, PRINTF_FORMAT const char* format, va_list args)
     buf.pid       = cfg.state.pidbuf;
     buf.name      = cfg.si.name;
 
+#if !defined(SIR_NO_TEXT_STYLING)
```

Comment:
we should verify that this is not making an assumption that `buf.style` will be filled with a valid string (or i suppose maybe it doesn't matter if that part is preprocessed out as well)

Reply:
I will check

Reply:
This is the only place I see buf.style actually